### ISSUE TITLE
Unable to load Bios [Android 5.1.1 Intel Atom Processor / NOT CRASHING]

### ISSUE THREAD

**Issue by MikeAW2010** [reicast/reicast-emulator#990]
I have my bios file stored on an external SD card, although I have tried on the internal SD and even in the reicast install directory and have the same results.

Reicast is properly pointed to the correct BIOS as it no longer tells me to set a path or the Bios. I also have Reicast installed on my phone and when I swap the SDCard to my phone, it loads the exact same Bios file without any problem, so I know its not the Bios.

I tried changing the area in the settings but it did not help.

What happens is this:

1. I press Boot BIOS
2. The screen turns black
3. If I press the back button I am able to see the dreamcast controls and settings on the bottom menu of the screen
4. Regardless if I press back or not, the screen stays black for 10 seconds then takes me back to the Reicast Browser Menu. 
5. The same thing happens if I try to play a game.

Basically the BIOS is not booting.

I've attached my Android information in these images:

http://i203.photobucket.com/albums/aa303/Need4Camaro/IMAG2430_zpsvntjwo1a.jpg

http://i203.photobucket.com/albums/aa303/Need4Camaro/IMAG2431_zpsptgycwcs.jpg

http://i203.photobucket.com/albums/aa303/Need4Camaro/IMAG2432_zpsh4ci7bqd.jpg

http://i203.photobucket.com/albums/aa303/Need4Camaro/IMAG2433_zpsjzeynvvg.jpg

Can anyone help me?

**Comment by gannon345** [comment]
I can't see the images, but generally this is because the BIOS isn't in a writable file, or is misnamed. BIOS needs to be on internal storage (because the app doesn't request permissions for external, so external is read only), and your bios files must be named DC_BOOT.bin and DC_FLASH.bin.

See if that helps.

**Comment by arutha13** [comment]
Is DC_BOOT.bin sould be uppercase ??? could explain my issue

**Comment by arutha13** [comment]
Hi,

I am trying to make reicast works on a nvidia tablet k1 with nougat and I have the same issue than above. 
All the bios are in internal storage and pointed to the right path, the bios are correctly named ... But only back screen ...  (I have try a lot of different bios from the web no one them solve my problem.
Any idea what else I can try to make it work ???

**Comment by LoungeKatt** [comment]
The android-studio changes specifically address issues with Android 7.0
